README.md

```diff
@@ -162,6 +162,7 @@ using the matched rule and runs it. Rules enabled by default are as follows:
 * `go_run` &ndash; appends `.go` extension when compiling/running Go programs
 * `grep_recursive` &ndash; adds `-r` when you trying to `grep` directory;
 * `gulp_not_task` &ndash; fixes misspelled gulp tasks;
+* `hadoop_dfs_missing_dash` &ndash; Add the missing dash to the command.
```

Comment:
Last comment and I stop annoying you for today: all the other items start with a verb conjugated at the 3rd person and with a lower case letter, and all but the last items end with a ';' not a '.'.